frontend/packages/console-shared/src/utils/firehose.ts

```diff
@@ -0,0 +1,39 @@
+import { FirehoseResult } from '@console/internal/components/utils';
+import { K8sKind, K8sResourceCommon, K8sResourceKind } from '@console/internal/module/k8s';
+import * as _ from 'lodash';
+
+export const isLoaded = <A extends K8sResourceCommon | K8sResourceCommon[] = K8sResourceKind[]>(
```

Comment:
I dont see a value in helpers such as these. Why not just do `result?.loaded` instead of calling this ?

Reply:
I agree with @rawagner - let's use the [optional chaining](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining) language feature introduced in TypeScript 3.7 to keep things short and simple.

We should also revisit existing helpers and, if possible, remove them in favor of using the optional chaining.

Reply:
ok, I agree for simple access like this it probably doesn't make sense, but for longer paths it is much more convenient to have a function for it - faster to read and less to prone to mistakes  when one selects a different field and doesn't notice it

keeping this helper in kubevirt for now